packages/orchestration/src/typeGuards.js

```diff
@@ -37,11 +37,17 @@ export const Proto3Shape = {
   value: M.string(),
 };
 
-// FIXME missing `delegatorAddress` from the type
+export const DenomShape = M.string();
+
+/** @type {TypedPattern<DenomAmount>} */
+export const DenomAmountShape = { denom: DenomShape, value: M.bigint() };
+
+export const AmountArgShape = M.or(NatAmountShape, DenomAmountShape);
```

Comment:
I think we're meant to support non-nat amounts, but the `orchestration-api.ts` is this narrow so this is consistent with that. FYI @dtribble 

Still I would suggest leaving this as `AmountShape` for future compatibility and so that we don't need a NatAmountShape` export from `@agoric/ertp`. It's better to use `makeNatAmountShape`, which we already have an export for, because it validates that particular brand. If people see the generic one they'll reach for that and in most cases they're better off with the narrower one.

Reply:
> For the OrchAccount exos, it seems we should still narrow to NatAmount 

Agree. But couldn't you just define that in the modules where you need instead of exporting it? e.g. 
```js
/** @see {makeNatAmountShape} to match a specific brand */
const AnyNatAmountShape = {brand: BrandShape, value: NatShape};
export const AmountArgShape = M.or(AnyNatAmountShape, DenomAmountShape);
```